src/renderer.ts

```diff
@@ -85,10 +86,10 @@ const bulletStyles = (colour: string): SerializedStyles => css`
 `;
 
 const HeadingTwoStyles = css`
-    font-size: 1.8rem;
-    line-height: 2.2rem;
-    margin: 1rem 0 4px 0;
+    font-size: 1.4rem;
     font-weight: 700;
+    margin: 1rem 0 4px 0;
```

Comment:
I'm not 100% sure `remSpace[4]` has been released yet @SiAdcock ?

```suggestion
    margin: ${remSpace[4]} 0 ${remSpace[1]} 0;
```